app/views/shared/_card.html.erb

```diff
@@ -3,22 +3,24 @@
     <div class="row g-0">
       <div class="col-md-4">
 
-        <%= cl_image_tag(activity.cl_tag, width: 150, height: 150, class: "card-position") %>
+        <%= cl_image_tag(activity.cl_tag, :format => :png, class: "card-img", transformation:[ {width: 245, height: 200, crop: "crop", gravity:"auto", quality: "auto"},
```

Comment:
this needs to be fixed - the transformation here is leading to an odd zoom in effect

Reply:
It's using AI to select the best part of the image to crop to as most of the images aren't very shrinkable. Played around quite a bit with them getting some to look nice as others were grainy or some other sort of artifacts. Will work it out